qutebrowser/mainwindow/tabwidget.py

```diff
@@ -501,8 +501,10 @@ def _minimum_tab_size_hint_helper(self, tab_text: str,
         """
         text = '\u2026' if ellipsis else tab_text
         # Don't ever shorten if text is shorter than the ellipsis
-        text_width = min(self.fontMetrics().width(text),
-                         self.fontMetrics().width(tab_text))
+        def _text_to_width(text):
```

Comment:
Add a blank line before this (or before the comment) to make flake8 happy :wink: